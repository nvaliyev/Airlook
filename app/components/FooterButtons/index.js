
import React from 'react';
import { Text, TouchableOpacity, } from 'react-native';
import styles from './style';

const FooterButtons = ({ onPress, ico, text }) => {
  return (
    <TouchableOpacity style={styles.footerButtons} onPress={onPress} >
      {ico}
      <Text style={styles.text}>{text}</Text>
    </TouchableOpacity>
  )
}
export default FooterButtons;