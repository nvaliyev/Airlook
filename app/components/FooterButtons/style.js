import { StyleSheet } from 'react-native';
import config from '../../servises/utils';

const styles = StyleSheet.create({

    footerButtons: {
        flexDirection: 'row',
        width: '50%',
        height: '100%',
        justifyContent: 'center',
        alignItems: 'center',
    },
    text:
    {
        color: config.lilac,
        fontSize: 16,
        marginHorizontal: 15,
    }


});
export default styles
