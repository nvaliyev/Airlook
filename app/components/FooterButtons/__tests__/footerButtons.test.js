import 'react-native';
import React from './node_modules/react';
import ShallowRenderer from './node_modules/react-test-renderer/shallow';
import FooterButtons from '..';
describe(' FooterButtons', () => {
    test(' FooterButtons snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(< FooterButtons/>);
        expect(snapshot).toMatchSnapshot();
    });
});


