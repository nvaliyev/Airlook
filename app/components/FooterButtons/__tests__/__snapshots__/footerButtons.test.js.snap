// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` FooterButtons  FooterButtons snapshot 1`] = `
<TouchableOpacity
  activeOpacity={0.2}
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
      "height": "100%",
      "justifyContent": "center",
      "width": "50%",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#F7F9FC",
        "fontSize": 16,
        "marginHorizontal": 15,
      }
    }
  />
</TouchableOpacity>
`;
